tests/tasks/setup/eks/awscli-cp.yaml

```diff
@@ -47,6 +47,9 @@ spec:
       
       if [ "$CREATED_CLUSTER" == "" ]; then
         aws eks create-cluster --name $(params.cluster-name) --region $(params.region) --kubernetes-version $(params.kubernetes-version) --role-arn $(params.servicerole) --resources-vpc-config subnetIds=$subnets,securityGroupIds=$sg $ENDPOINT_FLAG
+      else 
+        echo "cluster already exists; Aborting as cluster was not cleaned up already"
+        exit 1
```

Comment:
I don't think this is a good approach, we have a task to create a cluster with a name and if a cluster already exists it shouldn't fail just exit gracefully. 
This helps with testing and reiterating stuff as the cluster already exists and we can run tests again on this cluster.